packages/exoflex/example/patches/react-navigation-drawer+2.7.2.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/node_modules/react-navigation-drawer/lib/module/views/Drawer.js b/node_modules/react-navigation-drawer/lib/module/views/Drawer.js
+index 3f81450..cf976b7 100644
+--- a/node_modules/react-navigation-drawer/lib/module/views/Drawer.js
++++ b/node_modules/react-navigation-drawer/lib/module/views/Drawer.js
```

Comment:
After checking the issue, from my understanding it seems this was caused by we using react-navigation 4. It's not required to do this now but I think it's good to also upgrade it to react-navigation 5/6.

Reply:
Got it, will try and report later.

Reply:
Based on this: https://reactnavigation.org/docs/5.x/upgrading-from-4.x#navigation-container

I thinks it's better to upgrade on separate PR, since we still using `createAppContainer` to wrap our `RootNavigator`.